browser/js/w2ui-1.5.rc1.js

```diff
@@ -6423,6 +6428,219 @@ w2utils.event = {
                 // event after
                 this.trigger($.extend(edata, { phase: 'after' }));
             }
+        },
+        getDocType: function(map) {
+
+            var url = map.options.doc;
+            // Extract the file extension from the URL
+            var extension = url.substring(url.lastIndexOf('.') + 1);
+
+            // Convert the extension to lowercase
+            extension = extension.toLowerCase();
+
+            // Define the file type based on the extension using a switch case
+            switch (extension) {
+                case 'odt':
+                return 'text';
+                case 'odg':
+                return 'drawing';
+                case 'odp':
+                return 'presentation';
+                case 'ods':
+                return 'spreadsheet';
+                default:
+                return 'unknown';
+            }
+        },
+
+        createIconURL: function(cleanName, docType) {
+            var iconURLAliases = {
+                'copyformat' : 'formatpaintbrush',
```

Comment:
do we really need to copy paste here this list of aliases?
is that needed?

Reply:
yes because we are creating image url from object getting from Control.toptoolbar.js file.

 this object have some minor changes of name which is not same as image file name in images folder so better to map for now. 

we can change it that values in img object but that might be risky. that will effect on other parts of core code as well